cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForShuntCompensators.java

```diff
@@ -57,10 +57,15 @@ private void apply(ShuntCompensator shuntCompensator, CgmesRegulatingControlForS
         if (cgmesRc == null) {
             return;
         }
+        String rcId = cgmesRc.regulatingControlId;
+
+        // We are saving the relationship with the
+        // original regulating control even if it is not enabled
+        // or if we are not able to set it properly in IIDM
         if (!cgmesRc.controlEnabled) {
+            setAdditionalProperties(shuntCompensator, rcId);
```

Comment:
Not sure it is warranted. Is it useful to keep information that are not consistent? I understand it is useful for debug purposes but is there another usecase where this information is meaningful?

Reply:
We want to keep original regulating control information without changing the current logic about its use in IIDM.